src/doc/languagespec.tex

```diff
@@ -5223,7 +5223,7 @@ \subsection{Light emission closures}
 directly seen by the camera will directly reproduce the closure weight in the
 final pixel, regardless of being a surface or a volume.
 
-For surface emission, If you divide {\cf emission} by 
+For surface emission, if you divide {\cf emittance} by
```

Comment:
I think emission here refers to the closure.

Reply:
```suggestion
For surface emission, if you divide the return value of {\cf emission()} by
```

Reply:
Yeah, I think that removes some of the ambiguity